engine/CMakeLists.txt

```diff
@@ -1279,19 +1279,6 @@ ADD_SUBDIRECTORY("../doc" "./doc")
 # As per https://braintrekking.wordpress.com/2013/04/27/dll-hell-how-to-include-microsoft-redistributable-runtime-libraries-in-your-cmakecpack-project/
 INSTALL(FILES ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS} DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT system_libraries)
 
-IF (CMAKE_SYSTEM_NAME STREQUAL Windows)
-    FILE(GLOB_RECURSE OurDependencies FOLLOW_SYMLINKS LIST_DIRECTORIES false
-            ${CMAKE_SOURCE_DIR}/../*.dll
-            ${CMAKE_SOURCE_DIR}/../bin/*.dll
-            ${CMAKE_SOURCE_DIR}/*.dll
-            ${CMAKE_SOURCE_DIR}/bin/*.dll
-            ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/*.dll
-            ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/bin/*.dll
-            ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/${CPACK_BUILD_CONFIG}/*.dll
-            ${VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/${CPACK_BUILD_CONFIG}/bin/*.dll)
-    INSTALL(FILES ${OurDependencies} DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT system_libraries)
-ENDIF ()
```

Comment:
This is probably the bulk of the actual issue. We should have some targeted libraries we depend on, but that should be able to be explicit.

Reply:
Yeah, probably.